### PR TITLE
Several improvements for "uyuni-health-check"

### PR DESCRIPTION
**Pull request by meaksh**
This next round of improvements are coming from last Hackweek 22. There are several improvements here:

- Fix memory leak on `uyuni-health-exporter`.
- Fix python packaging and installation.
- Deploy grafana and prometheus dashboard.
- Fix loki and promtail deployments.
- Run all containers in the same POD.
- Unify console logging across deployment functions.
- More friendly CLI with new functions.
- Containers are not wiped by default after executions.
- Minor and cosmetic changes.
- Update README.md to reflect latest changes

### CODE REVIEW

File: src/uyuni_health_check/exporter/uyuni_health_exporter.py [hunk 1]
```diff
@@ -186,9 +244,21 @@ def collect(self):
...
     while True:
         # period between collection
         time.sleep(frequency)
         uyuni_data_gatherer.refresh()
+        gc.collect()
+        snapshot2 = tracemalloc.take_snapshot()
+        top_stats = snapshot2.compare_to(snapshot, "lineno")
+        print("[ Top 10 differences ]")
+        for stat in top_stats[:10]:
+            print(stat)
```
Comment (cbosdo) [inline src/uyuni_health_check/exporter/uyuni_health_exporter.py:260]: Isn't that a profiling tooling reminder to remove?

**Review by cbosdo — COMMENTED** [review]
Fantastic work. I just wonder what is the rational behind grouping everything in one pod? Is it to ease network setup between them?

File: src/uyuni_health_check/exporter/uyuni_health_exporter.py (hunk 1 above)
Comment (meaksh) [reply src/uyuni_health_check/exporter/uyuni_health_exporter.py:260]: Correct. Give me a sec to clean that up :)

**Comment by meaksh** [comment]
@cbosdo thanks for the review!

Yeah, the main reasons for grouping everything in one pod are:

- networking / dns: so I can configure i.a. grafana to have configured prometheus data source, or the prometheus one scraping the metrics from the `uyuni-health-exporter` containers easily.
- I can easily start/shutdown/clean the entire POD, with all containers.
- Somehow consistency around "uyuni-health-check" service. Everything for this service run in this pod.

**Review by cbosdo — APPROVED** [review]

**Pull request merged**